common/client-core/src/client/real_messages_control/mod.rs

```diff
@@ -205,7 +205,6 @@ impl RealMessagesController<OsRng> {
             stats_tx,
             shutdown_token.clone(),
         );
-
         RealMessagesController {
```

Comment:
I guess there was a `println` there? 
No harm in that but the whole change can be discarded, please undo that line deletion